package.json

```diff
@@ -8,7 +8,7 @@
   "scripts": {
     "prebuild": "npm run clean",
     "build": "gitbook build",
-    "clean": "del _book",
+    "clean": "rm -rf _book",
```

Comment:
If you do that, you should also remove `del-cli` from the `devDependencies`. But why remove it?

Based on https://github.com/openfisca/openfisca-doc/pull/53#issuecomment-297770903, you might be frustrated because tests fail on your machine. But have you run `npm install`?